packages/amplify-frontend-flutter/commands/flutter.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
What is this?

Reply:
This is a simple command to see whether the plugin exists with 'amplify flutter' in the terminal. It is present for all other frontend plugins. I'm not sure that it is really necessary, but is included for consistency.